<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>graphic</class>
 <widget class="QWidget" name="graphic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Graphic</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QCustomPlot" name="graph" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>601</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="xFrom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="xTo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>20</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="yFrom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="yTo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>50</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="fFrom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="fTo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>80</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
